app/views/LocationTracking.js

```diff
@@ -77,9 +78,22 @@ class LocationTracking extends Component {
         SetStoreData('PARTICIPATE', 'true').then(() =>
             LocationServices.start()
         );
-        this.setState({
-            isLogging:true
-        })
+
+        // Check and see if they actually authorized in the system dialog.
+        // If not, stop services and set the state to !isLogging
+        // Fixes tripleblindmarket/private-kit#129
```

Comment:
I personally don't include references to issues, I just explain the "why" like you did in the comment.  But it doesn't bother me too much.